core/modules/utils/dom/dragndrop.js

```diff
@@ -30,122 +30,130 @@ exports.makeDraggable = function(options) {
 	if(!options.selector && ((domNode.tagName || "").toLowerCase() !== "a")) {
 		domNode.setAttribute("draggable","true");
 	}
-	// Add event handlers
-	$tw.utils.addEventListeners(domNode,[
-		{name: "dragstart", handlerFunction: function(event) {
-			if(event.dataTransfer === undefined) {
-				return false;
+
+	var dragStartHandlerFunction = function(event) {
+		if(event.dataTransfer === undefined) {
+			return false;
+		}
+		// Collect the tiddlers being dragged
+		var dragTiddler = options.dragTiddlerFn && options.dragTiddlerFn(),
+			dragFilter = options.dragFilterFn && options.dragFilterFn(),
+			titles = dragTiddler ? [dragTiddler] : [],
+			startActions = options.startActions,
+			variables,
+			domNodeRect;
+		if(dragFilter) {
+			titles.push.apply(titles,options.widget.wiki.filterTiddlers(dragFilter,options.widget));
+		}
+		var titleString = $tw.utils.stringifyList(titles);
+		// Check that we've something to drag
+		if(titles.length > 0 && (options.selector && $tw.utils.domMatchesSelector(event.target,options.selector) || event.target === domNode)) {
```

Comment:
If possible we should split the line to make it a bit shorter and easier to read. eg:
```
if((titles.length > 0) && 
	(options.selector && $tw.utils.domMatchesSelector(event.target,options.selector) || 
	event.target === domNode)) {
```